plugin/src/main/java/at/helpch/chatchat/listener/PlayerListener.java

```diff
@@ -27,6 +28,15 @@ private void onJoin(final PlayerJoinEvent event) {
 
     @EventHandler
     private void onLeave(final PlayerQuitEvent event) {
+
+        // find everyone who last messaged the person leaving, and remove their reference
```

Comment:
This kinda sucks. Would it be better to do lazy? Not sure if it holds an actual Player reference, but if not, surely we have to look it up each time right? So could possibly be done lazily? Not sure but this kinda hurts